### DISCUSSION TITLE
A little political issue with China…

### DISCUSSION THREAD

**Discussion by shaucky** [airsdk/Adobe-Runtime-Support#2816] [General]
Hi.

I hope this doesn't feel inappropriate. The premise is that Chinese officials usually require Hong Kong, Macao and Taiwan to be referred to as regions, and China will block people from accessing websites that are illegal for China.
 
I noticed that on the AIR SDK order page, Hong Kong, Macao and Taiwan are country options. In order to avoid difficulties for Chinese mainland customers to access the AIR SDK website, i suggest making certain adjustments in accordance with the Chinese official intention.
 
In addition, the AIR SDK site footer seems to maintain the 2022 copyright notice?

**Comment by ajwfrost** [comment]
Hi

I believe we got the country list from some third party tool/component so will check what we can/should do there. It might be we have to separately list "mainland China" from other regions but will double-check. Thanks for raising it..

For the copyright, I'll get someone to look into it - it seems we should be probably having a date range now, 2019-2023...
https://www.thesmartbear.co.uk/copyright-your-website-content-the-right-way

thanks
